src/leap/mail/walk.py

```diff
@@ -19,18 +19,24 @@
 """
 import os
 
-from pycryptopp.hash import sha256
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes
 
 from leap.mail.utils import first
 
 DEBUG = os.environ.get("BITMASK_MAIL_DEBUG")
 
+
+def _get_hash(s):
+    digest = hashes.Hash(hashes.SHA256(), default_backend())
+    digest.update(s)
+    return digest.finalize().encode("hex").upper()
+
 if DEBUG:
     def get_hash(s):
-        return sha256.SHA256(s).hexdigest()[:10]
+        return _get_hash(s)[:10]
```

Comment:
it's a remnant from when this module was in development, it shows and print different parts of the message.